test/integration/zenodo_datapackage_test.py

```diff
@@ -8,24 +8,19 @@
 
 
 class TestZenodoDatapackages:
-    """Ensure production & sandbox Datastores point to valid datapackages."""
+    """Ensure all DOIs in Datastore point to valid datapackages."""
```

Comment:
It might be nice to also test that we can download a couple of the resources that are actually pointed at by the `datapackage.json`.

Reply:
It would, and we used to do this, but all of the "Actually download something from Zenodo" tests have ended up being so flaky that we've marked them `XFAIL` since they were routinely breaking the tests for no real reason.

Reply:
Yeah, I can definitely see how that would happen. Maybe someday we can use a "flaky test re-runner" like https://github.com/box/flaky.

Reply:
Something like that pytest flaky plugin seems like it would be good! But that one looks a bit stale.

IIRC much of the flakiness wasn't "re-run immediately and things are okay" it was more like "network is out for an hour so you'll just keep failing until it comes back" which seems weird given that GitHub and CERN should both be online kind of always but... 🤷🏼